src/System.Collections/tests/Generic/SortedSet/SortedSet.Generic.Tests.cs

```diff
@@ -283,4 +283,31 @@ public void SortedSet_Generic_CopyTo_NegativeCount_ThrowsArgumentOutOfRangeExcep
 
         #endregion
     }
+
+    public class SortedSet_ICollection_NonGeneric_Tests : ICollection_NonGeneric_Tests
+    {
+        protected override bool ICollection_NonGeneric_CopyTo_ArrayOfEnumType_ThrowsArgumentException { get { return true; } }
+        protected override bool Enumerator_Current_UndefinedOperation_Throws { get { return true; } }
+
+        protected override void AddToCollection(ICollection collection, int numberOfItemsToAdd)
+        {
+            int seed = numberOfItemsToAdd * 34;
```

Comment:
They're completely random. I needed a fixed seed so I mashed my face on the num-pad.